From mb@bu3sch.de  Thu Aug 13 16:52:39 2009
From: Michael Buesch <mb@bu3sch.de>
Date: Sun, 9 Aug 2009 18:02:30 +0200
Subject: Staging: rtl8187se: Fix comment-out-typo
To: gregkh@suse.de
Cc: linux-wireless@vger.kernel.org
Message-ID: <200908091802.31166.mb@bu3sch.de>
Content-Disposition: inline


This fixes a comment-out-typo that resulted in a dangling if-branch.

Signed-off-by: Michael Buesch <mb@bu3sch.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/staging/rtl8187se/r8180_core.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/rtl8187se/r8180_core.c
+++ b/drivers/staging/rtl8187se/r8180_core.c
@@ -2246,7 +2246,7 @@ short rtl8180_tx(struct net_device *dev,
 		// Use short preamble or not
 		if (priv->ieee80211->current_network.capability&WLAN_CAPABILITY_SHORT_PREAMBLE)
 			if (priv->plcp_preamble_mode==1 && rate!=0)	//  short mode now, not long!
-			//	*tail |= (1<<16);				// enable short preamble mode.
+			;//	*tail |= (1<<16);				// enable short preamble mode.
 
 		if(bCTSEnable) {
 			*tail |= (1<<18);
